pkg/rpc/client/rpc.go

```diff
@@ -597,3 +613,28 @@ func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, accs
 func (c *Client) GetNetwork() netmode.Magic {
 	return c.network
 }
+
+// GetNativeContractHash returns native contract hash by its name. It is not case-sensitive.
+func (c *Client) GetNativeContractHash(name string) (util.Uint160, error) {
+	lowercasedName := strings.ToLower(name)
+	switch lowercasedName {
+	case "neo", "gas":
+		if c.initDone {
+			return c.cache.nativeHashes[lowercasedName], nil
+		}
+
+	case "policy", "oracle", "designation":
```

Comment:
You're effectively duplicating native contract names here, while it's not really needed.
```go
hash, ok := c.cache.nativeHashes[lowercasedName]
if ok {
    return hash, nil
}
cs, err := c.GetContractStateByAddressOrName(lowercasedName)
...
```
If the name is invalid you'd get an error anyway.